Shop-1/src/controllers/products.ts

```diff
@@ -0,0 +1,110 @@
+import Product from "../models/Products";
+import Product from "../models/Product";
```

Comment:
duplicate import, you want the one with an s on the end. Match the file name in the model folder.